go/oasis-net-runner/fixtures/default.go

```diff
@@ -35,6 +41,28 @@ var (
 	keymanagerID common.Namespace
 )
 
+// GetFixture generates a fixture object from given file or default fixture, if no fixtures file provided.
+func GetFixture() (f *oasis.NetworkFixture, err error) {
+	if viper.IsSet(cfgFixtureFile) {
+		f, err = NewFixtureFromFile(viper.GetString(cfgFixtureFile))
+	} else {
+		f, err = NewDefaultFixture()
+	}
+	if err != nil {
+		return
+	}
+
+	// If requested, dump fixture to file and exit.
```

Comment:
This is so strange and looks like a hack, why would you use this in practice? Only if you want to dump the default fixture? If you want this you should instead add a sub-command that dumps the given fixture to stdout, e.g. `oasis-net-runner dump-fixture --fixture.default.node.binary /my/bin/oasis-node ...`.